fib/fib.py

```diff
@@ -0,0 +1,60 @@
+from timeit import timeit
+
+
+def fib(num):
+    if num == 0:
+        return 0
+    elif num == 1:
+        return 1
+    else:
+        return fib(num - 1) + fib(num - 2)
+
```

Comment:
nice use of recursion! I got this problem on the Hacker Rank test, and my approach was very... ahem... less elegant haha. I ended up making a list of elif cases for each iteration of the function from 1 to 10. It worked, but it wasn't my proudest moment! :)